src/VisualStudio/Core/Def/Telemetry/VSTelemetryActivityLogger.cs

```diff
@@ -17,7 +17,6 @@ internal sealed class VSTelemetryActivityLogger : ForegroundThreadAffinitizedObj
         private static readonly HashSet<FunctionId> s_functionIds
             = new HashSet<FunctionId>()
             {
-                FunctionId.NavigateTo_Search,
```

Comment:
this is perf and the other is information. if you don't use this, you won't be able to use common framework built in VS perf.